packages/code-utils/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "types": "./dist/src/index.d.ts",
   "dependencies": {
-    "cbor": "^5.1.0"
+    "cbor": "^5.2.0"
```

Comment:
was the minor bump here intentional?

Reply:
I mean, it wasn't *intentional* per se, but it wasn't unintentional either.  When I downgraded, I figured I might as well go to 5.2.0 (the latest 5.x) rather than what we had previously.

Reply:
Yeah, I suppose that's fine. I'll leave it up to you as to whether it's worth the effort to pursue further changes for 7.x, though by the fact that you didn't do that already I assume that you've already determined that it isn't.